### ISSUE TITLE
Err("Data not recordable") From data_request.py Example

### ISSUE THREAD

**Issue by billyen33** [nLabs-nScope/nscope-rs#22]
I tried running my `data_request.py` example from this repository with my nScope v1 and got the error seen in the title from line 13:

`data = nlab.read_all_channels(sample_rate, number_of_samples)`

I verified that my nScope is connected by running the list_all_nlabs.py example and also was able to successfully print out power from the monitor power example. This is the output from `list_all_nlabs.py`:

```
List of all detected nScopes:
Link to nLab v1 [ available: true ]
```

Is there a reason why this doesn't work? I used the example as-is and did not change anything. The error is:

`Err("Data not recordable") `

After this error the kernel just hangs here. I would like to access the nScope voltage readings via Python so I can write some additional code to use it as a logic analyzer.

**Comment by billyen33** [comment]
Any chance this would be updated? Tried multiple machines and the example still does not work.

**Comment by davidjmeyer** [comment]
Yes, sorry I missed this. You're right, it doesn't work on nScope v1, and actually worse than that it seems to hang forever.  The reason it doesn't work is because the communication bandwidth is lower on nScope v1 and it can't communicate back the entire request, so we fail to even initiate a request. However, the error message is not really helpful, and the program hangs.

Opened #24 to fix the issue.
